html/pfappserver/lib/pfappserver/PacketFence/Controller/Graph.pm

```diff
@@ -484,15 +485,29 @@ sub dashboard :Local :AdminRole('REPORTS') {
     $c->stash->{current_view} = 'HTML';
 }
 
-=head2 reports
+=head2 predefined_search
+
+Data to use for the predefined search
 
 =cut
 
-sub reports :Local :AdminRole('REPORTS') {
```

Comment:
Why rename the function as this will affect any references to it.
Either change the references to match the new name or rename it back to it's original name.